docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -658,6 +658,7 @@ Currently, you can only configure secondary environment if it uses BASIC authent
 | intervalTimeSettings.processes   | Low        | int     | 20000         | How often frontend reloads scenario list                                                      |
 | intervalTimeSettings.healthCheck | Low        | int     | 30000         | How often frontend reloads checks scenarios states                                            |
 | developmentMode                  | Medium     | boolean | false         | For development mode we disable some security features like CORS. **Don't** use in production |
+| enableConfigEndpoint             | Medium     | boolean | false         | Expose config over http (GET /api/app/config/) - requires admin permission                    |
```

Comment:
maybe let's add notice that it can expose passwords etc. ?

Reply:
done